package.json

```diff
@@ -21,6 +22,7 @@
     "react": "^18",
     "react-dom": "^18",
     "react-query": "^3.39.3",
+    "styled-components": "^6.1.12",
```

Comment:
🤔 why use styled components when we have tailwind in the project?

Reply:
also not being used, `#should` remove it